continuous_integration/Dockerfile

```diff
@@ -1,17 +1,7 @@
-FROM sequenceiq/hadoop-docker:2.7.1
+FROM mdurant/docker_images:hadoop
```

Comment:
It seems odd to me to use the version as a name.  What will you do if you need to update the version?  I suspect (but am not sure) that this goes against common naming conventions for docker containers.